code/sarix_model/plot_retro_sub.R

```diff
@@ -0,0 +1,53 @@
+library(hubValidations)
+library(hubVis)
+library(hubData)
+library(lubridate)
+
+ref_date <- as.Date("2024-01-06")
+
+locations <- read.csv("../../submissions-hub/auxiliary-data/locations.csv")
```

Comment:
Not sure who else might run this code, but if I understand R's `file.path()` correctly, it should ensure that these references work on Windows machines, etc (edit: I see you used it below--any reason why not here as well?)

Reply:
laziness 🙈